src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -144,6 +144,24 @@ export const fetchDictionaryConcepts = (
   dispatch(isFetching(false));
 };
 
+export const fetchConceptsByName = query => async (dispatch) => {
+  dispatch(isFetching(true));
+  const conceptType = localStorage.getItem('type');
+  const userTypeName = localStorage.getItem('typeName');
+  const dictionaryId = localStorage.getItem('dictionaryId');
+  const limit = 0;
+  const page = 1;
```

Comment:
what does page mean?

Reply:
@dkayiwa I thought it was some sort of backend implementation that page and limit needed to be passed on that URL (teams before had been doing so), but I have taken them out and the endpoint still works well.